### PR TITLE
Add simple tests

### PR DESCRIPTION
**Pull request by gfontenot**
For right now, it probably makes a reasonable amount of sense to test
this by comparing static input against static output. As more things
need to be parsed, we can just add them to the input and modify them to
show how we expect them to look in the output and the tests should fail.

### CODE REVIEW

File: bin/test [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```
Comment (gabebw) [inline bin/test:2]: `set -e`?

File: bin/test [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```
Comment (gabebw) [inline bin/test:1]: Document this script in the README?

File: bin/test [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+stack build
+
+stack exec reflow -- -w 78 --ignore-headers fixtures/input.txt > /tmp/output.txt
+
```
Comment (gabebw) [inline bin/test:8]: How about a test for reading on STDIN, too? You could add this at the end of this file:

``` sh
stack exec reflow -- -w 78 --ignore-headers < fixtures/input.txt > /tmp/output-stdin.txt
diff /tmp/output-stdin.txt fixtures/expected.txt
```

Comment (gfontenot) [inline bin/test:8]: My _main_ concern here is testing the parser/wrapper components. I'd rather not waste time running it twice. That being said, maybe this makes more sense as the way I should run the tests in general?

**Pull request merged**